.github/workflows/docs.yml

```
@@ -0,0 +1,65 @@
+name: docs
+
+on:
+  push:
+    branches:
+      - add_documentation
```

```suggestion
      - main
```
Using add_documentation branch is temp for testing